thesis-gui/src/components/HomePage.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from "react";
+import {
+    IonCard, IonCardContent, IonCardSubtitle, IonCardTitle,
+    IonContent,
+    IonLoading,
+    IonPage,
+    IonRouterLink, IonText
+} from "@ionic/react";
+import {useHistory} from "react-router";
+import {useCategoryContext} from "../context/CategoryContext";
+import {PageHeader} from "./common/PageHeader";
+
+const PAGE_NAME = 'Play';
+
+export const HomePage: React.FC = () => {
+    const { loadingState } = useCategoryContext();
+    const history = useHistory();
+
+    useEffect(() => {
+        if (loadingState === 'FAILURE') {
+            history.push('/error-page');
+        }
+    }, [loadingState, history])
+
+    return (
+        <IonPage>
+            <PageHeader name={ PAGE_NAME } />
+            <IonContent class="ion-padding">
+                <IonLoading isOpen={loadingState === 'LOADING'} />
```

Comment:
Same as above. I would not display loader for categories if we don't need them in this component.

Reply:
Done